src/clients/python/experimental_api_v2/library/__init__.py

```diff
@@ -105,5 +209,126 @@ def is_server_ready(self):
             If unable to get readiness.
 
         """
-        self.response = self.client_stub.get("/api/health/ready")
+        self.response = self._client_stub.get("/api/health/ready")
+        self._last_request_id = _raise_if_error(self.response['NV-Status'])
         return self.response.status_code == 200
+
+    def is_model_ready(self, model_name, model_version):
```

Comment:
model_version should be optional and default to -1, which means that the server will choose the version based on its version policy for the model.  Update docs and simple example appropriately.

Reply:
The api to retrieve model readiness with or without model version is a part of V2 right? Here as a workaround I am retrieving the model readiness by getting model_status first and then looking for the MODEL_READY value. I have made a note to update when integrating with HTTP V2.